### DISCUSSION TITLE
verifying boolean response body

### DISCUSSION THREAD

**Discussion by yinloo-ola** [getapid/apid#189] [General]
My API response returns a json as below
```json
{
  "success": true,
  "data": {
    "num": 2
  }
}
```
however when I use apid to verify the response
```jsonnet
      expect: {
        code: 200,
        headers: {
          'Content-Type': 'application/json; charset=UTF-8',
        },
        body: {
          success: true,
          data: {
            num: 2,
          },
        },
      },
```
i am getting **\`body.success\` is not false, got true**
May i know the correct way to draft this expect?

**Comment by yinloo-ola** [comment]
oh.. i saw that boolean match is not implemented..

**Reply by iv-p** [reply]
This is something I'm working on at the moment, expect it to be added very soon :) 

I've opened an issue  #190

**Reply by iv-p** [reply]
This has now been added in v0.7.2 :)
